docs/quick-start.md

```diff
@@ -116,21 +116,21 @@ The default configuration values are sufficient for most deployments. All option
 
 Kubewarden has three main components which you will interact with:
 
-* The PolicyServer
+* The [[< policy-server-component >]]
 * The ClusterAdmissionPolicy
 * The AdmissionPolicy
 
-### PolicyServer
+### [[< policy-server-component >]]
 
-A Kubewarden `PolicyServer` is managed by the `kubewarden-controller` and multiple `PolicyServers` can be deployed in the same Kubernetes cluster.
+A Kubewarden `[[< policy-server-component >]]` is managed by the `kubewarden-controller` and multiple `[[< policy-server-component >]]s` can be deployed in the same Kubernetes cluster.
 
-A `PolicyServer` validates incoming requests by executing Kubewarden policies against them.
+A `[[< policy-server-component >]]` validates incoming requests by executing Kubewarden policies against them.
 
-This is the default `PolicyServer` configuration:
+This is the default `[[< policy-server-component >]]` configuration:
 
 ```yaml
 apiVersion: policies.kubewarden.io/v1
-kind: PolicyServer
+kind: [[< policy-server-component >]]
```

Comment:
```suggestion
kind: PolicyServer
```

We shouldn't do it inside code blocks, as they are copy-pasted when trying examples.

Reply:
Oh, yep, they would not copy-paste from the markdown source.
I was on the impression that the substitution would be for a "PolicyServer" that is a link to a reference page or something. If that's the case, then the substition shouldn't be part of the yaml block I suppose. If not, disregard.

Reply:
Yeah, might be getting a little more complex. Would need variables policy-server-component and policy-server-component-in-a-code-block, or the variable replacement code needs to not add a link to a glossary if inside a code block. Hmmm.